### ISSUE TITLE
dataset EDA

### ISSUE THREAD

**Issue by TukBang** [TukBang/Pet-care-app#1]
Result (2023-03-13)
    - 레이블별 해상도 체크: 정형 데이터에서 볼 수 있듯, 모두 동일한 화질(FHD)이 아님 
    - 관심영역의 크기 224, 256, 384, 512로 학습 시도
        - 384부터 EfficientNetV2에서 batch size가 64정도 되면 그래픽카드 메모리 리소스 부족
        - ImageNet에서 가중치를 가져와, Fine-tuning을 시도하여 크기를 224로 고정
        0 전처리의 방법
          - 단순 관심영역 포함 cut: valid accuracy 약 50%, 빠른 Overfitting
          0.1. 관심영역만을 resize (linear 보간법 시도): 약 55%, 1번 방법보다는 늦은 Overffiting
            + 보간법을 BICUBIC(늘릴 때), AREA(줄일 때) 적용 시도 필요
          0.2. 관심영역을 비율에 따라 늘리거나 줄인 후 zero-padding (linear 보간법 시도): 약 60%, 1, 2번 방법보다는 늦은 Overfitting
            + 보간법을 BICUBIC(늘릴 때), AREA(줄일 때) 적용 시도 필요
    1. 너무 작은 width, height를 가지는 데이터들 처리 여부
      1.1. 너무 작은 데이터, 너무 큰 데이터의 분포 확인
      1.2. PCA, TSNE로 데이터의 모인 정도 확인
    2. 카메라 구도가 다를 때 사용하는 기술을 주제로 논문 찾기
    3. 털 영역 제거의 시도 필요

**Comment by TukBang** [comment]
EDA 분석 Document 작성 후 시나리오 테스트 중
